content/blog/python-none/index.md

```diff
@@ -0,0 +1,58 @@
+```markdown
+---
+{
+    title: 'Python None',
+    description: 'Interpreted Languages And Their Footguns',
+    published: '2022-07-27T20:00:00.945Z',
+    edited: '2022-07-27T20:00:00.945ZZ',
```

Comment:
We actually don't tend to fill out `edited` until such time as the blog post is edited. Might play havoc with automated bots that scrape our RSS feed:

```suggestion
    published: '2022-07-27T20:00:00.945Z',
```